charts/topolvm/values.yaml

```diff
@@ -161,7 +161,8 @@ lvmd:
 
   # lvmd.nodeSelector -- Specify nodeSelector.
   ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
-  nodeSelector: {}
+  nodeSelector:
+    instance.hetzner.cloud/is-root-server: "true"
```

Comment:
i guess you can remove this?

Reply:
@batistein I don't think this can be removed. Up to now we don't have a way to specifiy nodes which have storage.